src/styles/_colors.scss

```diff
@@ -26,6 +26,9 @@ $wxc-blue-90: #0A274A;
 $wxc-blue-95: #091A30;
 $wxc-blue-100: #07101C;
 
+$wxc-blue-40-alpha-30: rgba(100, 180, 250, .3);
```

Comment:
HEX values are used for all other colors, could you maintain consistency and convert this RGBA values?

Reply:
Right, I will convert